### ISSUE TITLE
[BUG] wallet balance shows zero

### ISSUE THREAD

**Issue by geekboood** [Chia-Network/chia-blockchain#7303]
In April, I mined 2 XCH and they are confirmed at 19xxxx height. However, when I import the mnemonics on anther computer, and both the wallet and the node are synced, my balance still shows zero. And I can confirmed that in the chiaexplorer website, my coin are not spent. The only different is that my receive address is changed. Can you help me with this problem?

**Comment by navyfeng** [comment]
same problem( version 1.2.11)

**Comment by radchenk0** [comment]
Hello!

Surprisingly, I've just found myself in the exactly same situation.

My story. Probably it will help to point out what I'm doing wrong.

A couple of weeks ago, I decided to jump in the project. I created virtual machine(Debian) on my laptop(laptop is on Windows), installed light wallet in it, bought a few xch on centralized exchange and sent them to my receive address that I got once installed light wallet.
Then I double checked that I got them and have access to my wallet by removing virtual machine, creating a new one and restoring wallet from mnemonic. Wallet was correctly restored and I saw my balance that I'd expected.

A couple of days ago, I downloaded the latest full node installer(1.2.11) on my desktop(running on Windows), finished installation, restored the wallet from my mnemonic and started waiting when it finishes sync process. It took 4-5 days(working hours) and now I cannot see my xch.
Confirm that node and wallet are synced with blockchain
Also confirm that coins are not stolen via xchscan.com

Can anyone help or suggest something?
Kind regards
